build/csharp_typemap.json

```diff
@@ -145,9 +145,20 @@
     "_LUID": "ValueTuple<ulong, long>",
     "_LARGE_INTEGER": "long",
     "IUnknown": "Silk.NET.Core.Native.IUnknown",
-    "tagRECT": "Silk.NET.Maths.Rectangle<long>",
-    "tagPOINT": "Silk.NET.Maths.Vector2D<long>",
-    "tagSIZE": "Silk.NET.Maths.Vector2D<long>",
+    "tagRECT": "Silk.NET.Maths.Rectangle<int>",
+    "tagPOINT": "Silk.NET.Maths.Vector2D<int>",
+    "tagSIZE": "Silk.NET.Maths.Vector2D<int>",
```

Comment:
breaking?

Reply:
Yes, but a bug fix. Our bindings literally won't work without this breaking change.

Reply:
Fair Enough